docs/resources/docker.md.erb

```diff
@@ -5,7 +5,7 @@ platform: linux
 
 # docker
 
-Use the `docker` InSpec audit resource to test configuration data for docker daemon. It is a very comprehensive resource. Please have a look at [docker_container](docker_container) and [docker_image](docker_image), too.
+Use the `docker` InSpec audit resource to test configuration data for the docker daemon. It is a very comprehensive resource. See also: [docker_container](docker_container) and [docker_image](docker_image), too.
```

Comment:
Could we change this to `Docker daemon`?

Reply:
done